.github/workflows/continuous-integration.yml

```diff
@@ -687,6 +703,14 @@ jobs:
             VERSION=${{ env.VERSION }}
           tags: ${{ steps.docker-meta-ubi9.outputs.tags }}
 
+      - name: Generate SBOM for image
+        uses: anchore/sbom-action@v0.17.9
+        with:
+          image: ${{ steps.docker-meta-ubi9.outputs.tags }}
+          artifact-name: cloudnative-pg-${{ env.VERSION }}-ubi9.spdx
+          upload-artifact: true
+          upload-artifact-retention: 7d
```

Comment:
We shouldn't have a retention for these artifacts, for any artifact that it's an SBOM

Reply:
This is the SBOM in CI workflow, which will be trigger and generate SBOM report for every ci run, probably a longer retention date ?